<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1385</width>
    <height>914</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 13pt &quot;URW Gothic&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="FindTriangle">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>780</y>
      <width>571</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Найти треугольник</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="Table">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>581</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>720</width>
      <height>720</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>720</width>
      <height>720</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>720</width>
      <height>720</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearGraph">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>670</y>
      <width>571</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Очистить графическое поле</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearTable">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>560</y>
      <width>571</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Очистить таблицу c точками</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>0</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Таблица координат&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeletePoint">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>320</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Удалить точку с номером:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="DelNumPointField">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>320</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Номер точки</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="AddXField">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>380</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddPoint">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>380</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Добавить точку:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="AddYField">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>380</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangePoint">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>440</y>
      <width>581</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Изменить координаты точки на новые:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="ChangeNumPointField">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>500</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Номер точки</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="UpdateXField">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>500</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Новый X</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="UpdateYField">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>500</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Новый Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="MouseCoordinatesLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>730</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="AnswerLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>770</y>
      <width>701</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1385</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Выход</string>
    </property>
    <addaction name="Quit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Информация</string>
    </property>
    <addaction name="AuthorInfo"/>
    <addaction name="TaskInfo"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="ClearAll">
   <property name="text">
    <string>Очистить все поля</string>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="Quit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="ClearAll_2">
   <property name="text">
    <string>Очистить все</string>
   </property>
  </action>
  <action name="Finding">
   <property name="text">
    <string>Найти четырехугольники</string>
   </property>
  </action>
  <action name="AuthorInfo">
   <property name="text">
    <string>Об авторе</string>
   </property>
  </action>
  <action name="TaskInfo">
   <property name="text">
    <string>Условие задания</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
